.vitepress/theme/index.ts

```diff
@@ -1,6 +1,7 @@
 // https://vitepress.dev/guide/custom-theme
 import { h } from 'vue'
 import Theme from 'vitepress/theme'
+import '../../src/assets/styles/main.scss'
```

Comment:
**Issue**: This breaks the dark theme. Seeing as you’re trying to isolate the components in an iframe, we probably don’t need this, right?

![image](https://github.com/kumahq/kuma-gui/assets/5774638/8288cd9f-5fcb-476e-8eb6-73b3224a7791)

Reply:
Resolving this following https://github.com/kumahq/kuma-gui/pull/1807#issuecomment-1828302119.